website/docs/docs/cloud/manage-access/about-access.md

```diff
@@ -8,142 +8,203 @@ pagination_prev: null
-more information.
+dbt Cloud users have both a license (individually or by group) and permissions (by group only) that determine what actions they can take. Licenses are account-wide, and permissions provide more granular access or restrictions to specific features.
 
-You can view the groups in your account or create new groups from the **Groups & Licenses**
-page in your Account Settings. <br />
+### Licenses
 
-<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/access-control/group-list-2.png" title="Viewing a list of groups in the Account Settings page." />
+Every user in dbt Cloud will have a license assigned. Licenses consume "seats" which impact how your account is [billed](/docs/cloud/billing), depending on your [service plan](https://www.getdbt.com/pricing).
 
-### SSO mappings
+There are three license types in dbt Cloud:
 
-SSO Mappings connect Identity Provider (IdP) group membership to dbt Cloud group
-membership. When a user logs into dbt Cloud via a supported identity provider,
-their IdP group memberships are synced with dbt Cloud. Upon logging in
-successfully, the user's group memberships (and therefore, permissions) are
-adjusted accordingly within dbt Cloud automatically.
+- **Developer** &mdash; User can be granted _any_ permissions.
+- **Read-Only** &mdash; User has read-only permissions applied to all dbt Cloud resources regardless of the role-based permissions that the user is assigned.
+- **IT** &mdash; User has [Security Admin](/docs/cloud/manage-access/enterprise-permissions#security-admin) and [Billing Admin](/docs/cloud/manage-access/enterprise-permissions#billing-admin) permissions applied, regardless of the group permissions assigned.
 
-:::tip Creating SSO Mappings
+Developer licenses will make up a majority of the users in your environment and have the highest impact on billing, so it's important to monitor how many you have at any given time.
 
-While dbt Cloud supports mapping multiple IdP groups to a single dbt Cloud
-group, we recommend using a 1:1 mapping to make administration as simple as
-possible. Consider using the same name for your dbt Cloud groups and your IdP
-groups.
+For more information on these license types, see [Seats & Users](/docs/cloud/manage-access/seats-and-users)
 
-:::
+### Permissions
+
+Permissions determine what a developer-licensed user can do in your dbt Cloud account. By default, members of the `Owner` and `Member` groups have full access to all areas and features. When you want to restrict access to features, assign users to groups with stricter permission sets. Keep in mind that if a user belongs to multiple groups, the most permissive group will take precedence.
+
+The permissions available depends on whether you're on an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) or [self-service Team](/docs/cloud/manage-access/self-service-permissions) plan. Developer accounts only have a single user, so permissions aren't applicable.
```

Comment:
The capitalization is inconsistent. I _think_ this is what we want.
```suggestion
The permissions available depends on whether you're on an [Enterprise](/docs/cloud/manage-access/enterprise-permissions) or [Self-service Team](/docs/cloud/manage-access/self-service-permissions) plan. Developer accounts only have a single user, so permissions aren't applicable.
```

Reply:
Is "Self-service" a proper noun here? I though the branding was just "Team"

Reply:
Ah gotcha! I think you're right!